docs: add ai best practices link to first-app tutorial intro (#62140)
angular/angular: adev/src/content/tutorials/first-app/intro/README.md

3. [WebStorm](https://www.jetbrains.com/webstorm/)
</docs-step>

<docs-step title="Optional: set-up your AI powered IDE">

> If you are following this tutorial in an AI powered IDE, [checkout Angular prompt rules and best practices](/ai/develop-with-ai).

Reviewer: ```suggestion
In case you're following this tutorial in your preferred AI powered IDE, [check out Angular prompt rules and best practices](/ai/develop-with-ai).
```

Reviewer: Keeping the language consistent across PRs.

> In case you're following this tutorial in your favorite AI powered IDE, check out our [prompt rules and best practicies](/ai/develop-with-ai).

Reviewer 2: ```suggestion
In case you're following this tutorial in your favorite AI powered IDE, check out Angular [prompt rules and best practicies](/ai/develop-with-ai).
```

> In case you're following this tutorial in your favorite AI powered IDE, check out Angular [prompt rules and best practicies](/ai/develop-with-ai).

Reviewer: If you are following this tutorial in an AI powered IDE, [checkout Angular prompt rules and best practices](/ai/develop-with-ai).
